tool/grind.dart

```diff
@@ -23,7 +23,11 @@ api() {
   Map<String, List<String>> usages = {};
 
   imports.split('\n').forEach((String line) {
-    if (line.trim().isEmpty) return;
+    if (line
```

Comment:
Feel free to omit braces for guard clauses. The new formatting is horrible.

Reply:
Yeah, this was a surprise to me.